cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -657,6 +657,104 @@ func TestApplyTo(t *testing.T) {
 	}
 }
 
+// TODO: this test having some conflict on feature gate:
```

Comment:
@siyuanfoundation  need your help here, running this test itself seems fine, but when running with the whole pkg, it got this panic.